cache_toolbox/core.py

```diff
@@ -42,53 +102,75 @@ def get_instance(model, instance_or_pk, timeout=None, using=None):
     """
 
     pk = getattr(instance_or_pk, 'pk', instance_or_pk)
-    key = instance_key(model, instance_or_pk)
-    data = cache.get(key)
 
-    if data is not None:
+    primary_model = model
+    descriptors = getattr(primary_model, '_cache_fetch_related', ())
```

Comment:
Should this be recursive? Probably. Might be worth ignoring that for the initial version though?